mlir/lib/Dialect/Rock/Generator/Conv2dGenerator.cpp

```diff
@@ -858,6 +841,23 @@ LogicalResult Conv2dGenerator::genConvModule(ModuleOp &module, int kernel_id,
   attributes.push_back(builder.getNamedAttr(
       "features", builder.getAttr<GemmFeaturesAttr>(config.features)));
 
+  auto paddingArray = SmallVector<int32_t, 4>{
```

Comment:
Is there any advantage to doing this style over `SmallVector<int32_t, 4> paddingArray = {...}`?